api/src/routing.js

```diff
@@ -655,6 +655,14 @@ app.all(
   dbDocHandler.request,
   authorization.setAuthorized // adds the `authorized` flag to the `req` object, so it passes the firewall
 );
+const initialReplication = require('./controllers/initial-replication');
+app.get(
+  '/initial-replication/get-ids',
```

Comment:
Let's version this to future proof it a little.

```suggestion
  '/api/v1/initial-replication/get-ids',
```